javaagent-tooling/build.gradle.kts

```diff
@@ -64,6 +63,21 @@ dependencies {
   testImplementation("com.google.guava:guava")
 }
 
+testing {
+  suites {
+    val testExceptionHandler by registering(JvmTestSuite::class) {
```

Comment:
does this target specific tests?

Reply:
Yes, just the `ExceptionHandlerTest` - I moved it to a separate source directory, along with all its dependencies. I found that when I run `./gradlew :javaagent-tooling:check` locally it was randomly failing on that test; probably because it interfered with other bytecode modifying tests in the `test` source root.

Reply:
oh yes, I missed that this is tied to directory name, thx!